### PR TITLE
chore: unity 2019.4.19f1 update

### PR DESCRIPTION
**Pull request by BrianAmadori**

### CODE REVIEW

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/SceneController.cs [hunk 1]
```diff
@@ -56,6 +56,8 @@ public interface ISceneController : IMessageProcessHandler, IMessageQueueHandler
         event SceneController.OnOpenNFTDialogDelegate OnOpenNFTDialogRequest;
     }
 
+    //
+
```
Comment (sandrade-dcl) [inline unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/SceneController.cs]: Can we remove this empty line? (I guess it was an oversight)

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/SceneController.cs [hunk 2]
```diff
@@ -68,6 +70,7 @@ public class SceneController : ISceneController
 
         public void Initialize(IRuntimeComponentFactory componentFactory)
         {
+            //
```
Comment (sandrade-dcl) [inline unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/SceneController.cs]: The same as above.

**Review by sandrade-dcl — CHANGES_REQUESTED** [review]
I have tested the build and it seems all the audio sfx (for example all the buttons clicks) have been lost. However the ambience sounds (the birds) are working.

**Comment by AjimenezDCL** [comment]
I'm having issues with the sound as well. In any case the rest looks good, I will drop an approve since Santi already requested changes. Good job!

**Review by AjimenezDCL — APPROVED** [review]

**Review by pravusjif — APPROVED** [review]
Tested the build with no problems besides the no-sfx issue

**Review by sandrade-dcl — APPROVED** [review]
Sound SFX fixed!

**Pull request merged**